web/src/lane-editor/EditWayControls.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+  import type { GeoJSON } from "geojson";
+  import Layer from "../osm2streets-svelte/Layer.svelte";
+  import { clickedLane, network } from "../osm2streets-svelte/store";
+  import AllEdits from "./AllEdits.svelte";
+  import Tags from "./Tags.svelte";
+
+  // TODO Is this layering and event plumbing nice?
+  let allEdits: AllEdits;
+
+  let way: bigint | null = null;
+  let gj: GeoJSON | null = null;
+
+  let layerStyle = {
+    type: "fill",
+    paint: {
+      "fill-color": "red",
+      "fill-opacity": 0.3,
+    },
+  };
+
+  $: {
```

Comment:
https://svelte.dev/tutorial/reactive-statements

This runs whenever anything referenced inside of here changes. In this case, `$clickedLane`. (The `$` is the syntax for accessing a store: https://svelte.dev/tutorial/auto-subscriptions)